app/views/jobs/show.php

```diff
@@ -22,84 +22,82 @@
+                            <div class="sidebar-text-info ml-10"><span class="text-description salary-icon mb-10">Lương</span><strong class="small-heading"><?= money_format($job->salary) ?></strong></div>
                         </div>
                         <div class="col-md-6 d-flex">
                             <div class="sidebar-icon-item"><img src="<?= asset('assets/imgs/page/job-single/experience.svg') ?>" alt="<?= env('APP_NAME') ?>"></div>
-                            <div class="sidebar-text-info ml-10"><span class="text-description experience-icon mb-10">Experience</span><strong class="small-heading">1 - 2 years</strong></div>
+                            <div class="sidebar-text-info ml-10"><span class="text-description experience-icon mb-10">Kinh nghiệm</span><strong class="small-heading">1 - 2 năm</strong></div>
                         </div>
                     </div>
                     <div class="row mt-25">
                         <div class="col-md-6 d-flex mt-sm-15">
                             <div class="sidebar-icon-item"><img src="<?= asset('assets/imgs/page/job-single/job-type.svg') ?>" alt="<?= env('APP_NAME') ?>"></div>
-                            <div class="sidebar-text-info ml-10"><span class="text-description jobtype-icon mb-10">Job type</span><strong class="small-heading">Permanent</strong></div>
+                            <div class="sidebar-text-info ml-10"><span class="text-description jobtype-icon mb-10">Công việc</span><strong class="small-heading">Dài hạn</strong></div>
                         </div>
                         <div class="col-md-6 d-flex mt-sm-15">
                             <div class="sidebar-icon-item"><img src="<?= asset('assets/imgs/page/job-single/deadline.svg') ?>" alt="<?= env('APP_NAME') ?>"></div>
-                            <div class="sidebar-text-info ml-10"><span class="text-description mb-10">Deadline</span><strong class="small-heading">10/08/2022</strong></div>
+                            <div class="sidebar-text-info ml-10"><span class="text-description mb-10">Ngày hết hạn</span><strong class="small-heading">10/08/2022</strong></div>
                         </div>
                     </div>
                     <div class="row mt-25">
                         <div class="col-md-6 d-flex mt-sm-15">
                             <div class="sidebar-icon-item"><img src="<?= asset('assets/imgs/page/job-single/updated.svg') ?>" alt="<?= env('APP_NAME') ?>"></div>
-                            <div class="sidebar-text-info ml-10"><span class="text-description jobtype-icon mb-10">Updated</span><strong class="small-heading">10/07/2022</strong></div>
+                            <div class="sidebar-text-info ml-10"><span class="text-description jobtype-icon mb-10">Ngày cập nhật</span><strong class="small-heading"><?php echo explode(" ", $job->updated_at)[0] ?></strong></div>
```

Comment:
Không sài `echo` keyword trong file view

```suggestion
                            <div class="sidebar-text-info ml-10"><span class="text-description jobtype-icon mb-10">Ngày cập nhật</span><strong class="small-heading"><?= $job->updated-at->format('d/m/Y') ?></strong></div>
```